mod_reforged/hooks/states/world_state.nut

```diff
@@ -13,4 +13,30 @@
 
     	return __original(_properties, _isPlayerInitiated, _isCombatantsVisible, _allowFormationPicking);
     }
+
+	q.onMouseInput = @(__original) function( _mouse )
+	{
+		local ret = __original(_mouse);
+		if (ret == false)	// If the original function wasn't able to process the mouseclick we try to do that with increased interaction range
+		{
+			local isEscorting = this.m.EscortedEntity != null && !this.m.EscortedEntity.isNull();
+			if (isEscorting && _mouse.getState() == 1 && !this.isInCameraMovementMode() && !this.m.WasInCameraMovementMode)
```

Comment:
The `isEscorting` local variable seems unnecessary. You can just put here `if (!::MSU.isNull(this.m.EscortedEntity) && `